package.json

```diff
@@ -21,7 +21,17 @@
     "easy-format": "https://github.com/npm-ml/easy-format.git#npm-1.2.0",
     "merlin-extend": "https://github.com/npm-ml/merlin-extend.git#npm-0.0.1"
   },
+  "exportedEnvVars": {
+    "PATH": {
+      "val": "./_build/bin/",
+      "resolveAsRelativePath": true,
+      "global": true,
+      "globalCollisionBehavior": "joinPath"
+    }
+  },
   "scripts": {
+    "whereisocamlc": ". dependencyEnv && which ocamlc",
+    "whereisreason": ". dependencyEnv && which refmt",
```

Comment:
When there is a collision on "PATH", does it join the local path to the beginning or to the end? If it is to the end, ocamlc and refmt will still use the globally installed one.